envoy/config/filter/network/mongo_proxy/v2/BUILD

```diff
@@ -0,0 +1,10 @@
+load("//bazel:api_build_system.bzl", "api_proto_library")
+
+licenses(["notice"])  # Apache 2
+
+api_proto_library(
+    name = "mongo_proxy",
+    srcs = ["mongo_proxy.proto"],
+    visibility = ["//envoy:friends"],
```

Comment:
Why does a network filter require such wide visibility?

Reply:
I changed it just `//docs` in the last update.